BaseLib/FileTools.cpp

```diff
@@ -39,28 +39,6 @@ bool IsFileExisting(const std::string &strFilename)
     return (stat (strFilename.c_str(), &buffer) == 0);
 }
 
-double swapEndianness(double const& v)
```

Comment:
This function is needed by the GocadSGridReader. Please, don't remove it.

Reply:
As discussed, there is a template version of this `T swapEndinness(T const&)`, and the code still compiles.
In the GocadGridReader two versions are called, one for doubles and another for floats, and implementations of both versions are different: float is written backwards [f3, f2, f1, f0], but double is split in two halves, which are written backwards each: [d3, d2, d1, d0, d7, d6, d5, d4].

Boost endian library handles only integers. A [Wikipedia article]( https://en.wikipedia.org/wiki/Endianness#Floating_point) on endianness says that the floating point numbers are treated as integers. The latter is contradicting the current implementation.

Dropping the change for now...